dlinfer/vendor/ascend/torch_npu_ops.py

```diff
@@ -341,3 +342,18 @@ def fused_attention(
                 attn_output,
             )
     return attn_output
+
+
+@register_ops(vendor_ops_registry)
+def weight_quant_matmul(
+    x: Tensor,
+    qweight: Tensor,
+    scale: Tensor,
+    offset: Optional[Tensor] = None,
+    bias: Optional[Tensor] = None,
+    group_size: Optional[int] = 0,
+):
```

Comment:
pls add description of this op

Reply:
I will add the description in the next commit.